### ISSUE TITLE
Debian repository invalid Packages files

### ISSUE THREAD

**Issue by jethrogb** [keybase/keybase-issues#2104]
The Debian repository at https://dist.keybase.io/linux/deb/repo/ contains invalid `Packages` files with an empty `Depends:` line:
```
Reading package lists... Error!
E: Problem parsing dependency Depends
E: Error occurred while processing keybase (NewVersion2)
E: Problem with MergeList /var/lib/apt/lists/dist.keybase.io_linux_deb_repo_dists_stable_main_binary-amd64_Packages
E: The package lists or status file could not be parsed or opened.
E: Problem parsing dependency Depends
E: Error occurred while processing keybase (NewVersion2)
E: Problem with MergeList /var/lib/apt/lists/dist.keybase.io_linux_deb_repo_dists_stable_main_binary-i386_Packages
E: The package lists or status file could not be parsed or opened.
```

**Comment by therealkbhat** [comment]
I can second this.
OS: Ubuntu 14.04.4 LTS

```text
E: Problem parsing dependency Depends
E: Error occurred while processing keybase (NewVersion2)
E: Problem with MergeList /var/lib/apt/lists/dist.keybase.io_linux_deb_repo_dists_stable_main_binary-amd64_Packages
E: The package lists or status file could not be parsed or opened.
```

**Comment by sbrl** [comment]
This also affects the Ubuntu 15.10 repos! I'm going to have to disable the repo temporarily in order to keep everything else up to date.

**Comment by tombrossman** [comment]
I can confirm this also affects me on 64-bit Ubuntu 15.10 with the same errors as above. The usual workaround is to delete the merge lists (assumes they are corrupted) and re-update: `sudo rm -vf /var/lib/apt/lists/* && sudo apt-get update` which does not resolve the issue. On my 64-bit machine the contents of `/var/lib/apt/lists/dist.keybase.io_linux_deb_repo_dists_stable_main_binary-amd64_Packages` is the following:

```
Package: keybase
Version: 1.0.14-0
Architecture: amd64
Installed-Size: 33876
Maintainer: Keybase Bugs <bugs@keybase.io>
Depends:
Priority: optional
Section: misc
Filename: pool/main/k/keybase/keybase_1.0.14-0_amd64.deb
Size: 5639838
SHA256: f58114299f06a8ddda009f8cb1b51f9d925c7eed40d0df2fee899352bfcce2af
SHA1: 326f8a901350fa3998c8f0cda34ab81abb45be85
MD5sum: 5a5748b57e9f4157e67863fa8d02ebe7
Description: Keybase's command line client
```
and `/var/lib/apt/lists/dist.keybase.io_linux_deb_repo_dists_stable_main_binary-i386_Packages` is this:
```
Package: keybase
Version: 1.0.14-0
Architecture: i386
Installed-Size: 28036
Maintainer: Keybase Bugs <bugs@keybase.io>
Depends:
Priority: optional
Section: misc
Filename: pool/main/k/keybase/keybase_1.0.14-0_i386.deb
Size: 5333430
SHA256: b4d18496c2f30b0dbce05ae2955bb2253028e92c014599fd09a40e6589ee4f79
SHA1: 9375e8eaf395900d656f7adb5d5a496b02e4dff0
MD5sum: 7958bf7d509325d506a82cd67af2c1eb
Description: Keybase's command line client
```
The most obvious difference between these and other package lists is that the "**Depends:**" contains no value. Spot-checking lists for other installed programs shows that they all list dependencies and none have that field blank. Hope that helps.

**Comment by aramboi** [comment]
Same issue here on Ubuntu Gnome 15.10 64bit during `sudo apt-get update`:
```
Reading package lists... Error!
E: Problem parsing dependency Depends
E: Error occurred while processing keybase (NewVersion2)
E: Problem with MergeList /var/lib/apt/lists/dist.keybase.io_linux_deb_repo_dists_stable_main_binary-amd64_Packages
E: The package lists or status file could not be parsed or opened.
```
Doing the usual `sudo rm -vf /var/lib/apt/lists/* && sudo apt-get update` does not solve the issue as @tombrossman pointed out. Might be that Keybase people are preparing for a new release and they messed up the update channel by mistake.

**Comment by maribert** [comment]
Same problem here. The temporary work-around is to disable the repository (comment out the "deb" line in the corresponding file in /etc/apt/sources.list.d -- keybase will keep working but is no longer kept current.

**Comment by sigvei** [comment]
Earlier versions have had ```Depends: libappindicator1```. I am also experiencing this bug.

**Comment by oconnor663** [comment]
This should be fixed in the latest 1.0.14-1 release. Sorry for the trouble. (Also reported at keybase/client/issues/2129)

**Comment by sbrl** [comment]
@oconnor663 Yep, that seems to have fixed it.
